application.conf.ctmpl

```diff
@@ -0,0 +1,44 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $dataSource := vault (printf "secret/dsde/firecloud/%s/hamm/dataSource" $environment)}}
+{{with $billing := vault (printf "secret/dsde/firecloud/%s/hamm/billing" $environment)}}
+{{with $googleProject := env "GOOGLE_PROJ"}}
+{{with $dnsDomain := env "DNS_DOMAIN"}}
+{{with $serviceVersion := env "SERVICE_VERSION"}}
+
+
+cromwell {
+  cromwellUrl = "https://cromwell1.{{$dnsDomain}}:443"
+}
+
+db {
+  default {
+    url = "{{$dataSource.Data.url}}"
+    user = "{{$dataSource.Data.user}}"
+    password = "{{$dataSource.Data.password}}"
+    poolInitialSize = 50
+    poolMaxSize = 100
+    poolConnectionTimeoutMillis = 5000
+    driver = "org.postgresql.Driver"
+  }
+}
```

Comment:
This is how scalike likes it's configs to look. Scalike supports Typesafe configs, which is used across all our repos, which is why I switched us back to using that instead of pureconfig

Reply:
pureconfig supports both typesafe config and camel case(https://github.com/pureconfig/pureconfig/issues/239#issuecomment-303574235). You wouldn't need to define those implicits manually except for some more custom ones. It'd also write unit tests for configurations

Reply:
I put those config items there because I think those are the ones we might end up tweaking. I think we should still use this config tool because that's what we use across our other repos and I don't see much of a reason to change it to pureconfig. We also haven't ever tested our configs

Reply:
I don't think never testing configs is a good tradition.

Reply:
Okay! We can decide that later :)